compiler/package.go

```diff
@@ -530,7 +566,7 @@ func translateFunction(typ *ast.FuncType, recv *ast.Ident, body *ast.BlockStmt,
 		// If a blocking function is being resumed, initialize local variables from the saved context.
 		localVarDefs = fmt.Sprintf("var {%s, $c} = $restore(this, {%s});\n", strings.Join(localVars, ", "), strings.Join(params, ", "))
 		// If the function gets blocked, save local variables for future.
-		saveContext := fmt.Sprintf("var $f = {$blk: "+funcRef+", $c: true, $r, %s};", strings.Join(c.localVars, ", "))
+		saveContext := fmt.Sprintf("var $f = {$blk: "+instanceVar+", $c: true, $r, %s};", strings.Join(c.localVars, ", "))
```

Comment:
nit: although this predates this PR, building a format string with string concatenation seems a bit odd, given that `instanceVar` doesn't contain formatting verbs.  :shrug:

Reply:
Good catch, done.